forem/forem
# Add snackbars to /tags

## What type of PR is this? (check all applicable)


## Description
Add snackbars to the /tags page when hiding and following tags:
<img width="1512" alt="Screenshot 2023-09-05 at 17 22 04" src="https://github.com/forem/forem/assets/2786819/f17f1f11-303e-435b-a179-a8320b33a68b">

Also add a snackbar if there is an error: ( I do a bit of error handling in this PR too based on [Add error handling to the tags page and the dashboard page#20050](https://github.com/forem/forem/issues/20050))
<img width="1512" alt="Screenshot 2023-09-05 at 22 11 40" src="https://github.com/forem/forem/assets/2786819/011f4147-8514-4fd8-b828-63db3e82b8db">

## Related Tickets & Documents

- Related Issue #https://github.com/forem/forem/issues/20042
- Related Issue #20050 [Add error handling to the tags page and the dashboard page#20050](https://github.com/forem/forem/issues/20050)
- Closes #20042 

## QA Instructions, Screenshots, Recordings

Hide and Follow tags on the /tags and see the snackbars pop up at the bottom: 
- Follow a tag, and you should see "You have followed <tag name>."
- Unfollow a tag, and you should see "You have unfollowed <tag name>."
- Hide a tag, and you should see "You have hidden <tag name>."
- Unhide a tag, and you should see "You have unhidden <tag name>."
- Mimic an error and you should see "An error has occurred."

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

      have not been included_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/l41Yq0LrP3i7GOpqw/giphy.gif)
